circuit/types/atomic_match.go

```diff
@@ -83,38 +83,21 @@ func SetAtomicMatchTxWitness(tx *AtomicMatchTx) (witness AtomicMatchTxConstraint
 	return witness
 }
 
-func ComputeHashFromAtomicMatchTx(tx AtomicMatchTxConstraints, nonce Variable, expiredAt Variable, hFunc MiMC) (hashVal Variable) {
+func ComputeHashFromAtomicMatchTx(api API, tx AtomicMatchTxConstraints, nonce Variable, expiredAt Variable, hFunc MiMC) (hashVal Variable) {
 	hFunc.Reset()
 	hFunc.Write(
-		tx.AccountIndex,
-		tx.BuyOffer.Type,
-		tx.BuyOffer.OfferId,
-		tx.BuyOffer.AccountIndex,
-		tx.BuyOffer.NftIndex,
-		tx.BuyOffer.AssetId,
-		tx.BuyOffer.AssetAmount,
-		tx.BuyOffer.ListedAt,
-		tx.BuyOffer.ExpiredAt,
+		PackInt64Variables(api, tx.AccountIndex, nonce, expiredAt, ChainId),
```

Comment:
Please confirm whether it is ok to fill in all 256 bits(exceed 254)

Reply:
fixed